src/test/java/gov/healthit/chpl/aqa/pageObjects/DpManagementPage.java

```diff
@@ -47,6 +51,25 @@ public static WebElement confirmPendingProductsLink(final WebDriver driver) {
         return driver.findElement(By.xpath("//*[@id=\"admin-nav.dpManagement.confirm\"]/a"));
     }
 
+    /**
+     * Find inspect button for uploaded listing to confirm.
+     * @param driver WebDriver
+     * @param testChplId is chpl id of listing to confirm
+     * @return the inspect button element
+     * @throws Exception if there is an exception
+     */
+    public static WebElement inspectButtonForUploadedListing(final WebDriver driver, final String testChplId) {
+        WebElement table = driver.findElement(By.id("pending-listings-table"));
+        ArrayList<WebElement> rows = (ArrayList<WebElement>) table.findElement(By.tagName("tbody")).findElements(By.tagName("tr"));
+        for (WebElement row : rows) {
+            ArrayList<WebElement> cols = (ArrayList<WebElement>) row.findElements(By.tagName("td"));
+            if (cols.get(0).getText().equalsIgnoreCase(testChplId)) {
```

Comment:
I like this, but think you should add an `ID_COLUMN` like the `ACTION_COLUMN`. I know checkstyle doesn't give us "magic number" errors for 0, 1, and maybe 2, but that's more to be nice to us then because they're not magic numbers.

Reply:
I'll make that update.